src/components/Filters/components/ConnectedFilterControl/tests/ConnectedFilterControl.test.tsx

```diff
@@ -202,6 +202,24 @@ describe('<ConnectedFilterControl />', () => {
       disabled: true,
     });
   });
+
+  it('does not render CenterContainer when no child element is "null"', () => {
+    const connectedFilterControl = mountWithAppProvider(
+      <ConnectedFilterControl>{null}</ConnectedFilterControl>,
+    );
+
+    expect(connectedFilterControl.find('.CenterContainer')).toHaveLength(0);
```

Comment:
One glorious day we'll complete the move from enzyme to `@shopify/react-testing` and we'll be able to write things like `expect(connectedFilterControl).not.toContainReactComponent('div', {className: 'CenterContainer'})` and it will be glorious.

Today is not that day though.

(Ignore this, nothing to do here, this is just me rambling)